package.json

```diff
@@ -102,5 +101,6 @@
     "url": "https://github.com/RetroAchievements/api-js/issues"
   },
   "homepage": "https://github.com/RetroAchievements/api-js#readme",
-  "author": "RAWeb Team"
+  "author": "RAWeb Team",
+  "packageManager": "pnpm@9.1.1+sha256.9551e803dcb7a1839fdf5416153a844060c7bce013218ce823410532504ac10b"
```

Comment:
Enforcing an exact SHA256 is probably too restrictive. How about we do:
```json
"packageManager": "^pnpm@9.1.1"
```

Reply:
I don't usually use the hash, but if you try by yourself the following:
corepack use pnpm@9.1.1

it's actually corepack who adds the hash

Reply:
Got it - I was not previously familiar with corepack. This makes sense to me.